### ISSUE TITLE
SSG build error on RHEL7 around gen_multiple_reference_tables.py

### ISSUE THREAD

**Issue by lenox-joseph** [ComplianceAsCode/content#7895]
#### Description of problem:
On RHEL7.9 with host python, the modified way of generating reference tables doesn't import things properly.
Commit that last touched the import is 2c3cdf3091d
```
[ 53%] Built target generate-rhel7-table-stig_per_profile_stig_gui
Scanning dependencies of target generate-general-ref-tables-rhel7
[ 53%] [rhel7-tables] generating HTML refs table for  anssi cis cui nist ospp pcidss references
Traceback (most recent call last):
  File "/tmp/SSGSRC.uKqp/content-master/utils/gen_multiple_reference_tables.py", line 11, in <module>
    from utils import gen_reference_table
  File "/tmp/SSGSRC.uKqp/content-master/utils/gen_reference_table.py", line 10, in <module>
    import tables.table_renderer
ImportError: No module named tables.table_renderer
make[2]: *** [tables/table-rhel7-anssirefs.html] Error 1
make[1]: *** [rhel7/CMakeFiles/generate-general-ref-tables-rhel7.dir/all] Error 2
make: *** [all] Error 2
```

#### SCAP Security Guide Version:
99a466d091775a42316cb95fc326b9eb600261e9

#### Operating System Version:
Red Hat Enterprise Linux 7.9

#### Steps to Reproduce:

1. Build SSG with Python 2.7.5 and python-jinja2 installed
2. Use switches: 
```
-DSSG_PRODUCT_DEFAULT=OFF -DSSG_PRODUCT_FIREFOX=ON 
-DSSG_PRODUCT_RHEL7=ON -DSSG_PRODUCT_RHEL8=ON -DSSG_PRODUCT_JRE=ON
```

#### Actual Results:
Build error
```
[ 53%] Built target generate-rhel7-table-stig_per_profile_stig_gui
Scanning dependencies of target generate-general-ref-tables-rhel7
[ 53%] [rhel7-tables] generating HTML refs table for  anssi cis cui nist ospp pcidss references
Traceback (most recent call last):
  File "/tmp/SSGSRC.uKqp/content-master/utils/gen_multiple_reference_tables.py", line 11, in <module>
    from utils import gen_reference_table
  File "/tmp/SSGSRC.uKqp/content-master/utils/gen_reference_table.py", line 10, in <module>
    import tables.table_renderer
ImportError: No module named tables.table_renderer
make[2]: *** [tables/table-rhel7-anssirefs.html] Error 1
make[1]: *** [rhel7/CMakeFiles/generate-general-ref-tables-rhel7.dir/all] Error 2
make: *** [all] Error 2
```

#### Expected Results:
SSG builds Firefox, RHEL7, RHEL8, JRE products.

#### Additional Information/Debugging Steps:
This doesn't fail with Python 3.6.8, so it's 100% a py2/py3 issue.

**Comment by Mab879** [comment]
Should be fixed by #7887. Is this working for you now?

**Comment by lenox-joseph** [comment]
@Mab879 CI seems to be fixed for CentOS7.
